### ISSUE TITLE
Unbound stops resolving DNS on connectivity loss

### ISSUE THREAD

**Issue by vaygr** [chriscrowe/docker-pihole-unbound#251]
If the container was started with no DNS resolution available (connectivity loss, etc.), Unbound doesn't resolve DNS, therefore pi-hole doesn't work, even if connectivity comes back up in a few minutes post-restart.

**Comment by graphixillusion** [comment]
I'm having the exact same error: did you find any solution to this?

**Comment by vaygr** [comment]
No, I didn't have a chance to troubleshoot this. I also think there are duplicate of this, as it seems #255 looks similar.

**Comment by graphixillusion** [comment]
I think it's a problem related to DNSSEC as i have these entries in the log:

reply error is SERVFAIL (EDE: DNSSEC bogus)

Now i've disabled it, let's try...
